servicetalk-opentracing-inmemory/src/main/java/io/servicetalk/opentracing/inmemory/DefaultInMemorySpanContext.java

```diff
@@ -16,54 +16,73 @@
 package io.servicetalk.opentracing.inmemory;
 
 import io.servicetalk.opentracing.inmemory.api.InMemorySpanContext;
-import io.servicetalk.opentracing.inmemory.api.InMemoryTraceState;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.Collections;
 import java.util.Map.Entry;
+import javax.annotation.Nullable;
 
 import static io.servicetalk.opentracing.inmemory.SingleLineValue.format;
-import static java.util.Objects.requireNonNull;
 
 /**
- * Span object used by the {@link DefaultInMemoryTracer}.
+ * SpanContext object used by the {@link DefaultInMemoryTracer}.
  */
-class DefaultInMemorySpanContext implements InMemorySpanContext {
+public class DefaultInMemorySpanContext implements InMemorySpanContext {
```

Comment:
can we make this class final for now (more restrictive access pattern to minimize our API surface)?